test/tests.bats

```diff
@@ -1243,27 +1260,12 @@ load 'test_helper/bats-assert/load'
   run /bin/sh -c 'cat ./config/postfix-virtual.cf | grep "test1@example.org test1@forward.com,test2@forward.com" | wc -l | grep 1'
   assert_success
 }
-
 @test "checking setup.sh: setup.sh alias del" {
-  echo -e 'test1@example.org test1@forward.com,test2@forward.com\ntest2@example.org test1@forward.com' > ./config/postfix-virtual.cf
-
+  echo 'test1@example.org test1@forward.com, test2@forward.com,' > ./config/postfix-virtual.cf
   ./setup.sh -c mail alias del test1@example.org test1@forward.com
-  run grep "test1@forward.com" ./config/postfix-virtual.cf
-  assert_output  --regexp "^test2@example.org +test1@forward.com$"
-
-  run grep "test2@forward.com" ./config/postfix-virtual.cf
-  assert_output  --regexp "^test1@example.org +test2@forward.com$"
-
   ./setup.sh -c mail alias del test1@example.org test2@forward.com
-  run grep "test1@example.org" ./config/postfix-virtual.cf
-  assert_failure
-
-  run grep "test2@example.org" ./config/postfix-virtual.cf
+  run cat ./config/postfix-virtual.cf | wc -l | grep 0
   assert_success
-
-  ./setup.sh -c mail alias del test2@example.org test1@forward.com
-  run grep "test2@example.org" ./config/postfix-virtual.cf
-  assert_failure
```

Comment:
Why are these lines removed?

Reply:
Thanks for reviewing. I have no idea, I was working on too branches at the same time.
:+1: